lib/elixir/lib/calendar.ex

```diff
@@ -70,6 +70,13 @@ defmodule Calendar do
   @callback leap_year?(year) :: boolean
 
   @doc """
+  Computes the day of the week from the specified `year`, `month`, and `day`.
+
+  Returns the day of the week as 1: Monday, 2: Tuesday, and so on.
+  """
+  @callback day_of_week(year, month, day) :: non_neg_integer()
```

Comment:
No. The return value is correct because this is meant to abstract multiple calendars (and not only ISO ones). Unless we are sure all supported calendars have exactly 7 days per week, then we should not change it.